_config.yml

```diff
@@ -138,7 +138,7 @@ nav_external_links:
 back_to_top: true
 back_to_top_text: "Back to top"
 
-footer_content: "Copyright &copy; 2015-2023 Varies per page, see headers. Documentation distributed by an <a href=\"https://github.com/open-horizon/open-horizon.github.io/blob/master/LICENSE\">CC-BY-4.0 license.</a>"
+footer_content: "Copyright &copy; 2015-2023 Varies per page, see headers. Documentation distributed by an <a href=\"https://github.com/open-horizon/open-horizon.github.io/blob/master/LICENSE\">CC-BY-4.0 license.</a> Open Horizon is a <a href= \https://wiki.lfedge.org/display/LE/Stage+2%3A+Growth\">Stage Two project</a> at <a href =\https://www.lfedge.org/\">LF Edge</a>."
```

Comment:
It looks like the beginnings of your hyperlinks are missing quote marks:

You have: `<a href= \` in two places.

It should be `<a href=\"` instead in both places.

Can you please fix and update the PR?

Reply:
Done!